dvc/scm/tree.py

```diff
@@ -57,9 +58,7 @@ def walk(self, top, topdown=True):
         def onerror(e):
             raise e
 
-        for root, dirs, files in os.walk(
+        for root, dirs, files in dvc_walk(
```

Comment:
Shouldn't we apply dvcignore logic to other trees as well?

Reply:
Meaning, we should probably ignore dvcignore'd paths when iterating over git tree too.